presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergTableLayoutHandle.java

```diff
@@ -35,13 +36,15 @@ public class IcebergTableLayoutHandle
         extends BaseHiveTableLayoutHandle
 {
     private final List<IcebergColumnHandle> partitionColumns;
+    private final List<Column> dataColumns;
```

Comment:
Can it be extracted from both HiveTableLayoutHandle and IcebergTableLayoutHandle to BaseHiveTableLayoutHandle?

Reply:
As discussed offline, it would require a relatively larger refactor to move Column from `presto-hive-metastore` to `presto-hive-common` module. So, we can keep this as-is for now unless there are some concerns.

Reply:
cc @tdcmeehan